File: PROGRAMMERS/12911.py
# 다음 큰 숫자
# lv 2

def solution(n):
    answer = 0
    tmp = n
    while answer == 0:
        tmp += 1
        if str(bin(n)).count('1') == str(bin(tmp)).count('1'):
            answer = tmp
    return answer

File: JAVASCRIPT/PROGRAMMERS/12911.js
// 다음 큰 숫자
// lv 2
// 문자열, 구현

function solution(n) {
  var answer = n;
  while (true) {
    answer++;
    if (
      n.toString(2).split('1').length === answer.toString(2).split('1').length
    ) {
      break;
    }
  }
  return answer;
}
